src/QueryBuilder/ResultViewer/helpers.js

```diff
@@ -14,10 +18,24 @@ export const getTableMetadata = (entityType) => {
   }, {});
 
   const defaultVisibleColumns = defaultColumns?.filter(col => col.selected).map(col => col.value) || [];
+  const formatter = defaultColumns.reduce((formatted, column) => {
+    const { value, dataType } = column;
+
+    formatted[value] = (item) => {
+      if (dataType === DATA_TYPES.DateType) {
+        return <FormattedDate value={item[value]} />;
+      } else {
+        return item[value] ? `${item[value]}` : '';
```

Comment:
Less repetition:
```
return `${item[value]}` || '';
```
If the string coercion is necessary, maybe add a comment to explain why so the next person who looks at this knows to leave it alone. Otherwise, this can be simplified further: 
```
return item[value] || '';
```

Reply:
Ye, looks better. Done! Thanks @zburke